ASSEMBLY.md

```diff
@@ -106,9 +108,9 @@ $ cargo install probe-run
 2. Ensure you have the Rust Programming Language installed (see <https://www.rust-lang.org>)
 3. Install the `probe-run` tool (if you didn't do it earlier):
 
-```console
-$ cargo install probe-run
-```
+	```console
```

Comment:
Unsure what this indentation is for?

Reply:
Elements in lists need to be indented according to https://www.markdownguide.org/basic-syntax/ under "Adding Elements in Lists"